### PR TITLE
[onert/lib] Move CopyInputInitializer class definition

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit get CopyInputInitializer out of RandomTestRunner.cpp like RandomInputInitializer

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Related issue: https://github.com/Samsung/ONE/pull/6049#discussion_r577284326

### CODE REVIEW

**Comment by seanshpark** [comment]
There's a conflict...

**Comment by hseok-oh** [comment]
I've resolved conflict

File: runtime/libs/tflite/src/CopyInputInitializer.cpp [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+{
+namespace tflite
+{
+
+void CopyInputInitializer::run(::tflite::Interpreter &interp)
+{
+  for (const auto &tensor_idx : interp.inputs())
+  {
+    TfLiteTensor *tensor = interp.tensor(tensor_idx);
+    if (tensor->type == kTfLiteInt32)
```
Comment (seanshpark) [inline runtime/libs/tflite/src/CopyInputInitializer.cpp:32]: how about using `switch-case` with `tensor->type`? As I remember, our static analysis tool reports less complicate score with `switch-case`

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Comment by seanshpark** [comment]
I've now noticed this PR is code relocation and changing the code wasn't necessary... My apologies to request such thing...

**Comment by hseok-oh** [comment]
>I've now noticed this PR is code relocation and changing the code wasn't necessary..

Runtime review process allows code relocation with small fix. So no problem :)

**Pull request merged**